package.json

```diff
@@ -860,6 +873,11 @@
                 }
             ],
             "commandPalette": [
+                {
+                    "command": "jupyter.notebookeditor.keybind.toggleOutput",
+                    "title": "%DataScience.toggleCellOutput%",
+                    "when": "notebookViewType == jupyter-notebook"
```

Comment:
Do we want this in the command palette since this command is supposed to be per cell?

Reply:
Obviously not, at first it wasn't clear if it was per cell or all the notebook.

Reply:
Let's add to command palette to have consistency with the other major keybindings. "Notebook: Toggle Output of Selected Cells"